ui/src/app/components/load-dump/load-dump.component.html

```diff
@@ -35,6 +35,17 @@ <h3 class="primary-header">Dump File</h3>
         >
         <mat-icon matSuffix *ngIf="uploadStart && uploadFail" class="danger">cancel</mat-icon>
       </mat-form-field>
+      <br>
+      <h3 class="primary-header">Spanner Dialect</h3>
+      <mat-form-field class="full-width" appearance="outline">
+        <mat-label>Spanner Dialect</mat-label>
+        <mat-select matSelect name="dialect" formControlName="dialect" appearance="outline">
+          <mat-option *ngFor="let element of dialect" [value]="element.value">
+            {{ element.displayName }}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+      <br>
```

Comment:
Can we declare this as a component and use it across `load-dump` and `direct-connect` instead of duplicating the code?

Reply:
based on the discussion going ahead with separate usage for now